generators/app/index.js

```diff
@@ -178,29 +179,51 @@ function handleAspNet(yo) {
     var aspNet = new AspNetHelper(baseImageName, portNumber);
 
     var done = yo.async();
-    aspNet.configureUrls(function (err, noteForUser) {
+    aspNet.updateProjectJson(function (err, projectJsonNote) {
         if (err) {
             error = true;
             yo.log.error(err);
+            done();
             return;
         }
-        configureUrlsNoteForUser = noteForUser;
-        done();
+        updateProjectJsonNoteForUser = projectJsonNote;
+
+        if (aspNet.getDotnetVersion() == 'RC2') {
+            aspNet.updateProgramCS(function (err, programCSNode) {
```

Comment:
Why is updateProgramCS under updateProjectJson? Shouldn't they be processed separately?